samples/boot/minimal/src/main/java/sample/Config.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sample;
+
+import org.springframework.boot.web.servlet.FilterRegistrationBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jose.jwk.JWK;
+import com.nimbusds.jose.jwk.JWKSet;
+import com.nimbusds.jose.jwk.KeyUse;
+import com.nimbusds.jose.jwk.gen.RSAKeyGenerator;
+
+@Configuration
+public class Config {
+
+    @Bean
+    FilterRegistrationBean<JWKSetEndpoint> registerJWKSetEndpoint() throws JOSEException {
+        FilterRegistrationBean<JWKSetEndpoint> fb = new FilterRegistrationBean<>();
+        fb.setFilter(new JWKSetEndpoint(generateJwkSet()));
```

Comment:
Let's register `JWKSetEndpoint` with the Spring Security filter chain via `HttpSecurity.addFilter...`